Source/WebKit/Shared/WTFArgumentCoders.serialization.in

```diff
@@ -20,7 +20,14 @@
 # OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 # OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
-webkit_platform_headers: "StreamConnectionEncoder.h"
+webkit_platform_headers: "StreamConnectionEncoder.h" "IPCValidation.h"
+headers: "IPCValidation.h"
+[WebKitPlatform, Alias=class WTF::String, Validator='IPC::Validation::isValidHostname($)'] alias Hostname;
+[WebKitPlatform, Alias=class WTF::String, Validator='IPC::Validation::isValidMimeType($)'] alias MimeType;
+[WebKitPlatform, Alias=class WTF::String, Validator='IPC::Validation::isValidURLScheme($)'] alias URLScheme;
```

Comment:
Will these be usable in `messages.in` too? I'm thinking about `WebProcess::RegisterURLSchemeAsEmptyDocument`, etc.